test/controllers/resource_controller_test.rb

```diff
@@ -191,4 +191,40 @@ class ResourceControllerTest < ActionDispatch::IntegrationTest
       post api_namespace_resource_index_url(api_namespace_id: api_namespace.id), params: payload
     end
   end
+
+  test 'should allow #create with the current_user & current_visit being available in the ' do
+    @api_namespace.api_form.update(properties: { 'name': {'label': 'name', 'placeholder': 'Test', 'type_validation': 'tel'}})
+
+    api_action = api_actions(:one)
+    api_action.update!(
+                      type: 'CreateApiAction',
+                      bearer_token: 'test-token',
+                      position: 0,
+                      action_type: 'send_web_request',
+                      request_url: "http://www.example.com/success",
```

Comment:
@alis-khadka I sort of get what this is testing, but we should have a sync about this test to make sure its not circular. 

not urgent, Happy to discuss during our weekly staff eng sync